recipe/meta.yaml

```diff
@@ -1,17 +1,18 @@
+{% set name = "ruamel.yaml" %}
 {% set version = "0.15.80" %}
 
 package:
-  name: ruamel_yaml
+  name: {{ name | lower | replace(".", "_") }}
```

Comment:
Is this actually a different package then?

Reply:
As I understand it, this is a mirror of the Anaconda Inc fork (ruamel_yaml) of the upstream project (ruamel.yaml). I'll hard-code the conda package name, but the upstream package is "ruamel.yaml".

Reply:
Right, but there is a separate `ruamel.yaml` package & feedstock for that

Reply:
Right. But there's also a `ruamel_yaml` package that's based on the same upstream Python code.

Reply:
To be honest, I made this PR before I understood the difference, but now I wonder whether to give up on the "ruamel_yaml" package entirely.

Reply:
Yeah, I think giving up on one of them is the right move, and I think it is this one.